go/enclave/vkhandler/vk_handler_test.go

```diff
@@ -34,7 +34,64 @@ func TestVKHandler(t *testing.T) {
 	assert.NoError(t, err)
 }
 
-func TestSignAndCheckSignature(t *testing.T) {
+func TestVKHandlerOGMessageFormat(t *testing.T) {
```

Comment:
Should this be a table test ? 
https://dave.cheney.net/2019/05/07/prefer-table-driven-tests